grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java

```diff
@@ -87,7 +88,7 @@
     private List<Query<?>> queries = null;
 
     GraqlSession(
-            Session session, GraknGraphFactory factory, String outputFormat,
+            Session session, GraknSession factory, String outputFormat,
```

Comment:
Yes I think we can reserve that specific change for later. Even the package names should change and I think that should be reserved for after the release